tests/unit/test_json.py

```diff
@@ -8,21 +8,36 @@
 
 class TestJson(unittest.TestCase):
     def setUp(self):
-        self.manifest_path = os.path.join(REPO_ROOT, "manifest.json")
+        self.manifest_path_v2 = os.path.join(REPO_ROOT, "manifest-v2.json")
 
-    def _load_manifest(self):
-        with open(self.manifest_path) as f:
+    def _load_manifest(self, manifest):
+        with open(manifest) as f:
             return json.load(f)
 
-    def test_manifest_valid_json(self):
-        json_body = self._load_manifest()
+    def test_manifest_v2_valid_json(self):
+        json_body = self._load_manifest(self.manifest_path_v2)
         self.assertIsNotNone(json_body)
 
-    def test_manifest_template_exist(self):
-        json_body = self._load_manifest()
-        for runtime, templates in json_body.items():
+    def test_manifest_v2_template_exist(self):
+        directories_dont_exist = []
+        json_body = self._load_manifest(self.manifest_path_v2)
+        for _, templates in json_body.items():
             for template in templates:
                 directory = template["directory"]
+                # use_case_name = template.get("useCaseName")
```

Comment:
nit: Should we remove this commented line?

Reply:
Updated.